src/util/integer_gmp_imp.h

```diff
@@ -78,6 +78,14 @@ class CVC4_PUBLIC Integer {
 
   ~Integer() {}
 
+  /**
+   * Returns a copy of d_value to enable public access of GMP data.
+   */
+  mpz_class get_mpz_copy()
+  {
+    return mpz_class(d_value.get_mpz_t());
```

Comment:
Does it suffice to just "return d_value;"?

Reply:
Indeed, it is. Renamed to getValue() and added a CLN implementation.